RevenueCatUI/Templates/V2/Components/Text/TextComponentView.swift

```diff
@@ -75,7 +75,7 @@ private struct NonLocalizedMarkdownText: View {
 
     var markdownText: AttributedString? {
         #if swift(>=5.7)
-        if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
+        if #available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *) {
```

Comment:
What about the macOS and tvOS changes? 👀

Reply:
if the availability matches the one of the struct we can probably drop it altogether?

Reply:
Yeah, realized the same and did it in https://github.com/RevenueCat/purchases-ios/pull/5267/commits/fdb46c4d2501c6a534c29e02d5ecb60cee435ca4
Thanks for pointing it out!